src/admin-views/dashboard/endpoints/endpoint.php

```diff
@@ -19,88 +19,109 @@
+	$queue_label   = _x( 'Queue', 'Label for the integration endpoint Dashboards endpoint queue.', 'event-automator' );
+	$queue_status  = _x( 'none', 'Label for the integration endpoint Dashboards endpoint queue status.', 'event-automator' );
+	if ( $endpoint['count'] > 0 ) {
+		$queue_status = _x( 'ready', 'Label for the integration endpoint Dashboards endpoint queue status.', 'event-automator' );
+	}
+	if ( $manager::$api_id === 'power-automate' ) {
+		$queue_status = $endpoint['count'];
+	}
 
-		if ( $endpoint['type'] === 'queue' && $endpoint['enabled'] ) {
-			$this->template( 'components/read-only', [
+	if ( $endpoint['type'] === 'queue' && $endpoint['enabled'] ) {
+		$this->template(
+			'components/read-only',
+			[
 				'classes_wrap'  => $queue_classes,
 				'label'         => $queue_label,
 				'screen_reader' => _x( 'The Queue for the integration endpoint.', 'The screen reader text of the label for the integration endpoint Dashboard endpoint queue.', 'event-automator' ),
-				'id'            => "tec_automator_integration_endpoint_queue_" . $endpoint['id'],
+				'id'            => 'tec_automator_integration_endpoint_queue_' . $endpoint['id'],
 				'name'          => "tec_automator_integration[]['endpoint_queue']",
 				'value'         => $queue_status,
-			] );
-		} else {
-			$this->template( 'dashboard/components/disabled', [
+			]
+		);
+	} else {
+		$this->template(
+			'dashboard/components/disabled',
+			[
 				'classes_wrap'  => $queue_classes,
 				'label'         => $queue_label,
 				'screen_reader' => _x( 'The Queue is disabled for this endpoint.', 'The screen reader text of the label for the integration endpoint Dashboard endpoint queue when disabled.', 'event-automator' ),
-			] );
-		}
+			]
+		);
+	}
 	?>
 	<div class="tec-automator-grid-item tec-settings-connection-endpoint-dashboard-details__actions-wrap tec-common-integration-endpoint-details__actions-wrap">
 		<?php
-			$this->template( 'dashboard/components/clear-button', [
-				'endpoint' => $endpoint,
-				'manager'  => $manager,
-				'url'      => $url,
-			] );
-			$this->template( 'dashboard/components/status-button', [
-				'endpoint' => $endpoint,
-				'manager'  => $manager,
-				'url'      => $url,
-			] );
-
-			if ( $endpoint['missing_dependency'] ) {
-				$this->template( 'dashboard/components/missing-dependency', [
+		if ( $endpoint['missing_dependency'] ) {
+			$this->template(
+				'dashboard/components/missing-dependency',
+				[
 					'endpoint' => $endpoint,
 					'manager'  => $manager,
 					'url'      => $url,
-				] );
-			}
+				]
+			);
+		} else {
+			$this->template(
+				'dashboard/components/clear-button',
+				[
+					'endpoint' => $endpoint,
+					'manager'  => $manager,
+					'url'      => $url,
+				]
+			);
+			$this->template(
+				'dashboard/components/status-button',
+				[
+					'endpoint' => $endpoint,
+					'manager'  => $manager,
+					'url'      => $url,
+				]
+			);
+		}
```

Comment:
If we're sending the same args into the template, it might be simpler to store them in a variable to pass them along:
```
$template_args = [
	'endpoint' => $endpoint,
	'manager'  => $manager,
	'url'      => $url,
];
```

That way, the template lines are on one line:
```
$this->template( 'dashboard/components/missing-dependency', $template_args );
```